### ISSUE TITLE
Sylph generates empty files when no genome is found

### ISSUE THREAD

**Issue by fplazaonate** [bluenote-1577/sylph#32]
Hi @bluenote-1577!

I have used sylph to generate species abundance profiles of many metagenomic samples.
I run the 'profile' command on each sample separatly and then I use a R script to merge the results and create the abundance table.
However, for some samples with a very low number of reads, sylph detects no genome and generates an empty file.
I makes my script crash as it expects at least one header.
Would it be possible to systematically write the header?

I have dug into the code and I think removing the following condition will do the trick?
https://github.com/bluenote-1577/sylph/blob/475b2bd417b5dd12aec5a880d5e62b12796c840a/src/contain.rs#L333C25-L333C38

**Comment by bluenote-1577** [comment]
Hi Florian, this seems like a reasonable request. I forgot why I set the program to only write header when a genome is present. I agree that _always_ writing a header would work better.

I've implemented a simpler change which simply removes mutex stuff and do `print_header` before iteration. I've pushed this change to the main branch. Will release a conda update in the near future.

BTW: I removed native compile by default in v0.7 so sylph should be installable via conda for your system now (i recall you had an older CPU).
